### PR TITLE
Add sliderule to environment

### PR DESCRIPTION
**Pull request by scottyhq**
Hi! It would be great to add sliderule python client (https://slideruleearth.io/web/) in advance of https://icesat-2-2023.hackweek.io

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -107,6 +107,7 @@ dependencies:
 
   # Packages specific to cryo work
   - icepyx==0.7.0
+  - sliderule==3.5.0
```
Comment (tsutterley) [inline environment.yml:110]: `sliderule` uses `ipyleaflet` 0.17.3 for some visualization (REMA/ArcticDEM). might be good to bump that up as well.

Comment (weiji14) [reply environment.yml:110]: Yep, and we're moving towards compatible release pins `~=` now for non CalVer packages per #46.
```suggestion
  - sliderule~=3.5.0
```

**Review by weiji14 — APPROVED** [review]
:rocket:

**Pull request merged**